smurfs/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import "./App.css";
+import { connect } from "react-redux";
+import Smurf from "./components/Smurf"
+ import SmurfForm from "./components/SmurfForm";
+import * as actions from "./state/actions";
+import styled from "styled-components";
+
+function App({
+  formValues,
+  changeInput,
+  fetchSmurfs,
+  smurfs,
+  postSmurf
+}) {
+
+  useEffect(() => {
+    fetchSmurfs();
+  }, []);
+
+  return (
+    <StyledApp className="App">
+      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQ6N9Fe6bOm5AWmq09craEMvH1nlzQOSKSSwRG6a8Y7qIo_7m1N" alt=""/>
```

Comment:
The image link seems a bit long to put in this. To make it better let's try to keep it in a separate file.